orders/views.py

```diff
@@ -1,3 +1,42 @@
-from django.shortcuts import render
+import json, uuid
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+
+from orders.models    import OrderStatus, Order, OrderItem
+from users.models     import User
+from products.models  import Product
+
+from users.utils      import user_decorator
+
+class CartView(View):
+    @user_decorator
+    def post(self, request):
+        try:
+            data           = json.loads(request.body)
+            user           = request.user
+            product        = data['product_id']
+            current_amount = 0
+
+            if not Product.objects.filter(id=product).exists():
+                return JsonResponse({'message':'PRODUCT_DOES_NOT_EXIST'}, status=400)
+
+            order, created = Order.objects.get_or_create(
+                        user            = user,
+                        order_status_id = OrderStatus.PENDING,
+                        defaults        = {'order_number':uuid.uuid4()}
+                    )
+
+            if OrderItem.objects.filter(order=order, product_id=product).exists():
+                current_amount = OrderItem.objects.get(order=order, product_id=product).amount
+                
+            orderitem, created = OrderItem.objects.update_or_create(
```

Comment:
```suggestion
            orderitem, is_created = OrderItem.objects.update_or_create(
```
여기도 boolean을 갖기 때문에 다음과 같이 수정하면 좋을 것 같습니다.

Reply:
네 수정하겠습니다!